### ISSUE TITLE
Disco API doesn't contain Liberica Native 22.2 and 21.3.3

### ISSUE THREAD

**Issue by strizhik** [foojayio/discoapi#63]
Hi! 

Looks like https://api.foojay.io/disco/v3.0/distributions doesn't contain 22.2.0 and 21.3.3 for Liberica Native:

`{
"name": "Liberica Native",
"api_parameter": "liberica_native",
"maintained": true,
"official_uri": "https://bell-sw.com/",
"synonyms": [
"liberica_native",
"LIBERICA_NATIVE",
"libericaNative",
"LibericaNative",
"liberica native",
"LIBERICA NATIVE",
"Liberica Native"
],
"versions": [
"22.1",
"22.0.0.2",
"21.3.2",
"21.3.1",
"21.3",
"21.2",
"21.1",
"21.0.0.2"
]
},`

**Comment by HanSolo** [comment]
Will directly check it, thx for the heads up

**Comment by HanSolo** [comment]
You are right these pkgs seems to be missing, will fix it and will let you know as soon as the fix will be in production.

**Comment by HanSolo** [comment]
Fixed the problem and deployed it to production, it might take up to 24h before the new pkgs will appear.

**Comment by sdeleuze** [comment]
I see the package in https://api.foojay.io/disco/v3.0/packages?distro=liberica_native now, thanks for the fix.

**Comment by strizhik** [comment]
Thanks for the fix.
